### ISSUE TITLE
Save/Cancel button location

### ISSUE THREAD

**Issue by wszarmach** [krico/jas#435]
These buttons are at the top of the screen but when I am creating or editing something I start at the top and move to the bottom then have to scroll back up to select one of the buttons.
Should these perhaps be at the bottom of the screen?

**Comment by pauljanes** [comment]
Yes good point. I had also meant to raise this as an issue. Only an usual person would start bottom and work upwards.

Agree with Waldemar that they should be at bottom

**Comment by pauljanes** [comment]
Unusual person i meant

**Comment by dragonflypl** [comment]
There are different approaches. Generally UX research says that if users don't see "Save" (action) buttons when they access page, it causes "friction" . If form in very long, you have to scroll down to save it. Which is not bad for adding new items, but for editing, not necessarly

**Comment by pauljanes** [comment]
And what about top and bottom for the Edit screens?

**Comment by dragonflypl** [comment]
Suggestion: if form is small show bottom or top buttons (so user does not see duplicated actions)
